data/json/achievements.json

```diff
@@ -2202,5 +2216,19 @@
         "description": "Reach a Mine"
       }
     ]
+  },
+  {
+    "id": "achievement_reach_police_station",
+    "type": "achievement",
+    "name": "I shot the Sheriff!  What?!  He was a zombie!",
```

Comment:
But this is not an achievement for killing a zombie cop though

Reply:
Zombie cops are a dime a dozen and spawn everywhere. It makes more sense to lock the profession behind the location rather than zombie cops.

Reply:
Yes, I was more refering to the joke that's not so much related to the actual achievement

Reply:
Ah I see your point. I may have to think of another description of the achievement and alter it in the next round of achievement additions.

Reply:
I'm not sure what you mean by next round, please fix it in this PR.

Reply:
Changed to "Zombie PD, to protect and zombify!"